app/views/govuk_publishing_components/components/docs/inverse_header.yml

```diff
@@ -0,0 +1,41 @@
+name: Inverse header
+description: A wrapper to contain header content in white text
+body: |
+  This component can be passed a block of template code and will wrap it in a blue header. This is as light-touch as possible and doesn't attempt to deal with the margins and paddings of its content, neither does it enforce the white text it requires. Implemented to accomodate topic and list page headings and breadcrumbs but unopinionated about its contents.
+
+accessibility_criteria: |
+  The component must:
+
+  * be used in conjunction with content that renders a text contrast ratio higher than 4.5:1
+  against the header colour to meet WCAG AA.
+
+examples:
+  default:
+    data:
+      block: |
+        <div class="pub-c-title pub-c-title--inverse">
+          <h1 class="pub-c-title__text ">
+            HTML publication page title
```

Comment:
Can we change the examples to Topic/List page examples to reflect how we think this component will be used at the moment

Reply:
Do you mean the content text? Or the (rendered) component html?

Reply:
Just changing 'HTML publication page title' to (for example) 'Education, training and skills' to show how it looks on a Topic page. When topic pages are live, we can link to an example in context, but for now we can do this as a quick win to give an example of what it would look like.